### PR TITLE
add the possibility to update a loader group directly from the interface

### PR DESCRIPTION
**Pull request by ydelattre**

### CODE REVIEW

File: grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/serviceLogic/UiV2Group.java [hunk 1]
```diff
@@ -4203,5 +4204,51 @@ public void groupEditCompositeSubmit(HttpServletRequest request, HttpServletResp
       GrouperSession.stopQuietly(grouperSession);
     }
   }
+  
+  /**
+   * update a loader group
```
Comment (mmoayyed) [inline grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/serviceLogic/UiV2Group.java]: Could you describe a bit more, what this method actually is doing? what exceptions may be thrown and when, etc?

File: grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/beans/api/GuiGroup.java [hunk 2]
```diff
@@ -648,4 +651,34 @@ public boolean isGrantAllAttrRead() {
...
+    //On vérifie que l'attribute loader LDAP est bien affecté
+    if(grouperLoaderLdapName != null) {
+	  hasAttrLdap = this.group.getAttributeDelegate().hasAttribute(grouperLoaderLdapName);
+    }
+    return hasAttrLdap;
+  }
+
+  /**
+   * if an attribute GrouperLoader SQL is assigned to this group
+   * @return true
```
Comment (mmoayyed) [inline grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/beans/api/GuiGroup.java]: Doesnt seem correct. You are also returning false, right? Can you document the method behavior?

File: grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/beans/api/GuiGroup.java [hunk 3]
```diff
@@ -648,4 +651,34 @@ public boolean isGrantAllAttrRead() {
   public boolean isGrantAllAttrUpdate() {
     return this.group.hasGroupAttrUpdate(SubjectFinder.findAllSubject());
   }
+  
+  /**
+   * if an attribute GrouperLoaderLdap is assigned to this group
+   * @return true
```
Comment (mmoayyed) [inline grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/beans/api/GuiGroup.java]: See my comment below please.

**Comment by mmoayyed** [comment]
I see a number of comments in the code are written in French, right? Could we have them in English please?

**Comment by mmoayyed** [comment]
Also, the PR does not seem to want to compile:
https://travis-ci.org/Internet2/grouper/jobs/57782428

**Comment by ydelattre** [comment]
i tried to follow your comments.
Is it OK ?
Thanks :)

**Comment by mmoayyed** [comment]
Much better, thank you so much.

**Comment by mchyzer** [comment]
I took your code and tweaked it a little bit and added to 2.2

https://bugs.internet2.edu/jira/browse/GRP-1131

**Pull request closed without merging**